chain/consensus/mir/manager.go

```diff
@@ -48,12 +48,14 @@ var (
 
 // Manager manages the Lotus and Mir nodes participating in ISS consensus protocol.
 type Manager struct {
-	// Lotus related types.
-	NetName dtypes.NetworkName
-	Addr    address.Address
-	Pool    *fifo.Pool
+	GetValidators validator.MembershipReader
```

Comment:
Sanity-check: This is not used anywhere, right?

Reply:
You are right